stats/player_game_log.py

```diff
@@ -73,7 +70,14 @@ def fetch_season(self, season_id):
         for row in player_info:
             # Checking matchup for home team.
             if '@' in row[9]:
-                self.game_set.add((season_int, row[7], row[8], row[9]))
+                if row[10] == "W":
```

Comment:
This bit of logic deserves UTs. It's a bit hard to write UTs when code isn't setup for packaging.

Something to think of if you are interested long term.

Reply:
Sorry, what is UT? Unit Test?

Reply:
I am a bit fuzzy on this logic. Does this have the potential two `GameEntry`s per game?

Reply:
Yeah UT = Unit Tests. Most of the requesters are just taking the response data and feeding into the table, so I don't see too much benefit.

The game table, and now the logic for this custom, so it's best to have a UT.

> I am a bit fuzzy on this logic. Does this have the potential two GameEntrys per game?

Damn, you are right, there would be two entries. I didn't realize it because I didn't see duplicates in the final table. I believe it's because `game_id` is PrimaryKey.

I suppose this is fine, but I can see if the logic can be cleaned. What do you think?

Reply:
On thinking further, no, there shouldn't be two game entries because, there are two kinds of entries in the resutlSet of PlayerGameLogs

1. `"CLE @ WAS"`
2. `"WAS vs. CLE"`

So the second one will never get picked up because this path only goes through when ` @ ` is present.

Reply:
Ah yes good catch. Wow, I should have left a comment about that.

Reply:
Oh, I did `# Checking matchup for home team.` isn't descriptive enough to describe what I meant, apparently.